tests/integration/kubernetes/gha-run.sh

```diff
@@ -433,7 +433,7 @@ function cleanup_nydus_snapshotter() {
 	fi
 	sleep 180s
 	kubectl delete -f "misc/snapshotter/nydus-snapshotter-rbac.yaml"
-	kubectl get ns nydus-system -o json | jq '.spec.finalizers = []' | kubectl replace --raw "/api/v1/namespaces/nydus-system/finalize" -f || true
+	kubectl get namespace nydus-system -o json | jq 'del(.spec.finalizers)' | kubectl replace --raw "/api/v1/namespaces/nydus-system/finalize" -f - || true
```

Comment:
hi @GabyCT !

The change of `jq '.spec.finalizers = []'` to ` jq 'del(.spec.finalizers)'` is really needed to? I'm just worried that some environments might have different versions of `jq` and this `del` syntax might not be available.

you know, `yq` left some bad marks on my soul...so I am always worried about the backward-compatibility of tools like `jq` too.

Reply:
@wainersm I tried it with the `jq '.spec.finalizers = []' but it did not work so seems like jq '.spec.finalizers = []' it is needed to properly delete the namespace

Reply:
ah, ok. thanks for the explanation @GabyCT !